sdk/inc/azure/core/az_mqtt5_rpc.h

```diff
@@ -20,6 +20,41 @@
 
 #include <azure/core/_az_cfg_prefix.h>
 
+/**
+ * @brief The default topic format for making RPC requests.
+ */
+#define AZ_MQTT5_RPC_SERVER_DEFAULT_SUBSCRIPTION_TOPIC_FORMAT \
```

Comment:
I this is used for the client and the server, should the naming be more generic? (same for others)